<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>www</class>
 <widget class="QWidget" name="www">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPlainTextEdit" name="fsck">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>291</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button_send">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="name_text">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>100</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>输入姓名</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_confirm">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="send_text">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>140</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>输入私信好友姓名</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="neiro">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>291</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="tongzh">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="iip_text">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>104</width>
     <height>21</height>
    </rect>
   </property>
   <property name="plainText">
    <string>IP：</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="pport_text">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>104</width>
     <height>21</height>
    </rect>
   </property>
   <property name="plainText">
    <string>PORT：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
